napi.h

```diff
@@ -142,6 +142,26 @@ static_assert(sizeof(char16_t) == sizeof(wchar_t),
     }                                                                          \
   } while (0)
 
+#define NAPI_INTERNAL_STRINGIFY_(x) #x
```

Comment:
It seems not being used. Did you intend to use it in the code below in places like `#expr`?

Reply:
Yes, that part was removed. Updated.